apps/src/aiEvaluation/studentWorkEvaluationsApi.ts

```diff
@@ -24,19 +24,24 @@ export async function logStudentWorkEvaluations(
   });
   // For each specific skill-based evaluation, log a UserLevelSkillEvaluation
   if (parsedResponse.skillEvaluations) {
-    parsedResponse.skillEvaluations.forEach((skillEvaluation: AIResponse) => {
-      logStudentWorkEvaluation({
-        type: 'UserLevelSkillEvaluation',
-        studentId: studentWorkSample.studentId,
-        levelId: levelId,
-        unitId: unitId,
-        evaluator: 'AI',
-        evaluationCriteria: skillEvaluation.evaluationCriteria,
-        evaluation: skillEvaluation.aiEvaluation,
-        reasoning: skillEvaluation.aiReasoning,
-      });
-    });
-    // TODO: Log an EvaulationSummary associating the UserLevelSkillEvaluation with the UserLevelEvaluation
+    parsedResponse.skillEvaluations.forEach(
+      async (skillEvaluation: AIResponse) => {
```

Comment:
I feel like I have gotten feedback before that `async` isn't needed inside a `forEeach`... I need to do more deep reserarch/understanding on that... flagging for now, BUT I will go do more research on this too=.

Reply:
🤔 Maybe you got feedback about something like this: https://medium.com/@potatoscript/exploring-the-difference-between-using-async-await-in-a-for-loop-and-foreach-739c9ebeb64a? In this case, the order of execution doesn't matter; but, you're right I was essentially throwing away the promises. I think to leverage calling these in parallel, we want to use Promise.all and then map based on this [article](https://sentry.io/answers/are-there-any-issues-with-using-async-await-with-foreach-loops-in-javascript/). Does that sound right to you?